packages/plugin/src/sibling-operations.ts

```diff
@@ -60,32 +60,30 @@ export function getSiblingOperations(options: ParserOptions, gqlConfig: GraphQLC
 
   // We first try to use graphql-config for loading the operations paths, based on the type of the file,
   // We are using the directory of the file as the key for the schema caching, to avoid reloading of the schema.
-  if (options && options.filePath && !options.skipGraphQLConfig) {
+  if (gqlConfig && options?.filePath) {
```

Comment:
Same here, by checking for `options.skipGraphQLConfig` and `gqlConfig` is redundant.

Reply:
Please see my comment above, we need to have the ability to explicitly skip graphql-config.